src/taxonomy/data/api.js

```diff
@@ -7,6 +7,7 @@ const getApiBaseUrl = () => getConfig().STUDIO_BASE_URL;
 export const getTaxonomyListApiUrl = (org) => {
   const url = new URL('api/content_tagging/v1/taxonomies/', getApiBaseUrl());
   url.searchParams.append('enabled', 'true');
+  url.searchParams.append('page_size', '500'); // For the tagging MVP, we don't paginate the taxonomy list
```

Comment:
@ChrisChV FYI - I included a quick fix here for the pagination issue on the taxonomy list page.